### ISSUE TITLE
Error on attempting to build

### ISSUE THREAD

**Issue by gregorypenn** [hadley/ggplot2-book#122]
I'm attempting to build the pdf of the book and I'm encountering an error from the R code. I've included the entire build log below, because it's short.

```
mkdir -p book/tex
mkdir -p book/tex/_figures
mkdir -p book/tex/diagrams
cp book/ggplot2-book.tex book/tex/ggplot2-book.tex
Rscript book/render-tex.R data-manip.rmd
Rscript book/render-tex.R ggplot.rmd
Loading required package: nlme

Attaching package: 'nlme'

The following object is masked from 'package:dplyr':

    collapse

This is mgcv 1.8-12. For overview type 'help("mgcv-package")'.
Rscript book/render-tex.R introduction.rmd
Quitting from lines 180-182 (introduction.rmd)
Error in desc$Priority : $ operator is invalid for atomic vectors
Calls: <Anonymous> ... <Anonymous> -> structure -> package_info -> vapply -> FUN
Execution halted
make: *** [book/tex/introduction.tex] Error 1
```

I know what "$ operator is invalid for atomic vectors" means, but I can't figure out where this is coming from. In fact, I grepped through the whole source directory for "desc$Priority" and couldn't find that string anywhere.

In what may be a related issue, I may not have all the R dependencies installed. Specifically, `devtools::install_deps()` returns an error about the package "root" not being found and I haven't been able to find it on my own either.

**Comment by knbknb** [comment]
Another reader of the book here.

I am affected by ~10 similar errors as well. I just commented a few blocks in each .Rmd file out. It was okay, only little info/ context was lost, a little plot here and there. 

In your case, I'd go to the R code block in line  180-182 in file `introduction.Rmd` and add  `eval=FALSE` to the block containing the error, then run `make` again to get it transpiled to a .tex file. Then a pdf will be created with a few plots missing, but you can work with this.

**Comment by kaemo** [comment]
Try:

``` r
install.packages("BH")
```

It worked for me.

**Comment by ohad** [comment]
I needed to also 
`install.packages("broom")`

**Comment by plutext** [comment]
installing BH and broom got me past that error.  On to the next one...

**Comment by hadley** [comment]
Should be fixed by #142
